eval.cpp

```diff
@@ -784,6 +793,11 @@ namespace Sass {
       Expression* ex = operator()(var);
       return evacuate_quotes(interpolation(ex));
 
+    } else if (Parent_Selector* var = dynamic_cast<Parent_Selector*>(s)) {
+
+      Expression* ex = operator()(var);
```

Comment:
If it can be done so safely please to. I dig into why the above `operator()` call was done that. 
I'm a big stickler for conventions.